default.nix

```diff
@@ -0,0 +1,27 @@
+let
```

Comment:
@QAston do we need the `default.nix` file in here?

Reply:
no, we can delete it